source/enumerate-collections.rst

```diff
@@ -433,6 +440,9 @@ The shell implements the first algorithm for falling back if the
 
 Version History
 ===============
+Version 0.6.2 Changes
+    - Support ``authorizedCollections`` option in ``listCollections`` command.
```

Comment:
The previous patch version bump to 0.6.1 made sense because it was renaming `slaveOkay` to `secondaryOk`.

In this case, you're introduce a new option to the spec. That seems more like a minor version bump, so I'd suggest 0.7.0 in this case.